documentation/source/install/openshift.rst

```diff
@@ -0,0 +1,273 @@
+:orphan:
+
+Running OpenEBS on a Multi-Node CentOS 7 OpenShift Cluster
+------------------------------------------------------------
+
+This section provides detailed instructions on how to setup a multi-node Bring-Your-Own-Host (BYO) OpenShift cluster on CentOS 7 and run applications on it with OpenEBS storage.
+
+Pre-requisites:
+^^^^^^^^^^^^^^^^^^
+* At least 2 or more CentOS 7 hosts (virtual-machines/baremetal/cloud instances) with 2 vCPUs, 4G RAM and 16GB hard disk.
+
+* Ensure that the following package dependencies are installed on the hosts using yum install. 
+  **Note:** yum update may be needed prior to this step.
+
+  -   python, wget, git, net-tools, bind-utils, iptables-services, bridge-utils, bash-completion, kexec-tools, sos, psacct, docker-1.12.6
```

Comment:
Lets include `iscsi-initiator-utils` package as well - so it will be 

python, wget...docker-1.12.6, iscsi-initiator-utils

Reply:
Done!